### ISSUE TITLE
history.yml doesn't loading if default path has been changed

### ISSUE THREAD

**Issue by unlok** [Shopify/dashing#583]
Overriding in `config.ru` occurs after loading on the default path? It is bug?
https://github.com/Shopify/dashing/blob/master/lib/dashing/app.rb#L39

**Comment by unlok** [comment]
I added to `config.ru` set about it:
```ruby
configure do
  set :history_file, File.expand_path('../data/dashboard/history.yml')
end
```

```ruby
puts settings.history_file # -> history.yml

if File.exists?(settings.history_file)
  set history: YAML.load_file(settings.history_file)
else
  set history: {}
end
```
but saving occurs right.

**Comment by tylermauthe** [comment]
We set the history file on line 32 of app.rb:

https://github.com/Shopify/dashing/blob/937fcce2dcca8010dbe8e9e26df954cb1330c1e0/lib/dashing/app.rb#L32

Not sure if I would consider this a bug, but it would be quite easy to fix.

Should be a sensible addition as long as it is added as an optional configuration parameter.

**Comment by unlok** [comment]
@tylermauthe 
Yes we set occurs on line 32, but on line 39 we immediately read by default path `history.yml`, but not from config `../data/dashboard/history.yml`.

And the result is that write to file occurs by `../data/dashboard/history.ym` path but reading by `./history.yml` path. 

In my opinion this is bug.

I fixed this in fork, but i had split `config.ru` file into two - `boot.ru` and `config.rb`.
https://github.com/unlok/dashing/commit/441f989cef6a66cfa0184bf9e86463c926498dc7

`config.rb` contains only the config section:
```ruby
configure do
  set :history_file, File.expand_path('../data/dashboard/history.yml')
end
```
`boot.ru` contains all the rest.

Сan you offer more correct approach?
